README.md

```diff
@@ -1,22 +1,17 @@
 [![Netlify Status](https://api.netlify.com/api/v1/badges/23b098eb-c146-4dea-96ed-287909bcda39/deploy-status)](https://app.netlify.com/sites/cherryontech/deploys)
 
-# Welcome To Cherry On Tech's website
+# 🎉 Welcome to Cherry On Tech's Website! 🌟
 
-We are a tech squad! We support each other as we learn and grow in our tech careers. We got the idea from [Mo Hampton](https://twitter.com/MoxieHampton)'s [JuneteenthConf](https://www.juneteenthconf.com/) talk, "[Breaking into Tech](https://www.youtube.com/watch?v=2N5fI5j-IIs)." Learn more about us in an upcoming blog post!
+We are a **tech squad** dedicated to supporting each other as we learn and grow in our tech careers. Our inspiration comes from [Mo Hampton](https://twitter.com/MoxieHampton)'s insightful talk at [JuneteenthConf](https://www.juneteenthconf.com/), titled "[Breaking into Tech](https://www.youtube.com/watch?v=2N5fI5j-IIs)." Stay tuned for an upcoming blog post to learn more about us!
 
-## Tech Stack
+## 🛠️ Tech Stack
 
-This website is built using [Gridsome](https://gridsome.org/), a static site generator based on [VueJS](https://vuejs.org/). It's under pretty heavy construction, but we love it already, and we hope you do, too!
+This website is built using [Gridsome](https://gridsome.org/), a static site generator based on [VueJS](https://vuejs.org/). While it's still under construction, we love it already, and we hope you do too! ❤️
 
-## Installation
+## 🚀 Installation
 
-1. Make sure you have [node](https://nodejs.org/) installed. You can check by running `node --version`
-1. Install Gridsome CLI tool if you don't have it: `npm install --global @gridsome/cli` or `yarn global add @gridsome/cli`
-1. Clone the repo: `git clone https://github.com/cherryontech/website.git`
-1. `cd website`
-1. `npm install` or `yarn install`
-1. Run `gridsome develop` to start a local dev server. In the terminal, you will see an address such as `http://localhost:8080/`. Navigate to that address in a browser to see the project in action.
```

Comment:
Issue: The installation steps are important to remain here. If you'd like to enhance this section by moving the node check instructions (as you added on ln15), feel free to do that.